addons/test_mail/tests/test_mail_gateway.py

```diff
@@ -56,6 +56,19 @@ def test_message_parse_eml(self):
         # Test that the parsing of mail with embedded emails as eml(msg) which generates empty attachments, can be processed.
         mail = self.format(test_mail_data.MAIL_EML_ATTACHMENT, email_from='"Sylvie Lelitre" <test.sylvie.lelitre@agrolait.com>', to='generic@test.com')
         self.env['mail.thread'].message_parse(email.message_from_string(pycompat.to_text(mail)))
+    def test_message_parse_eml_bounce_headers(self):
```

Comment:
This is bothering me:
```suggestion

    def test_message_parse_eml_bounce_headers(self):
```

:smile:

Reply:
This is why I didn't notice it :smile: 

![image](https://user-images.githubusercontent.com/186131/100619592-b0e81500-331d-11eb-9873-5062be768d48.png)